packages/common/src/index.ts

```diff
@@ -849,6 +853,22 @@ export function itemsInMetricQuery(
           ];
 }
 
+function formatRawValue(
+    field: Field | Metric | TableCalculation | CustomDimension,
+    value: any,
+) {
+    const isTimestamp =
+        isField(field) &&
+        (field.type === DimensionType.DATE ||
+            field.type === DimensionType.TIMESTAMP);
+
+    if (isTimestamp) {
+        // We want to return the datetime in UTC to avoid timezone issues in the frontend like in chart tooltips
+        return dayjs(value).utc(true).format();
```

Comment:
previous moment implementation 

> return moment(value).utc(true).format();

Reply:
Looks the same, no? It seems to work? I'm happy to test again after standup.